[0.045s] Invoking command in '/root/researchv2/build/diagnostics': CMAKE_PREFIX_PATH=/root/researchv2/install/self_test:/root/researchv2/install/diagnostic_updater:/root/researchv2/install/diagnostic_aggregator:/root/researchv2/install/urg_node:/root/researchv2/install/urg_node_msgs:/root/researchv2/install/urg_c:/root/researchv2/install/diagnostics:/root/researchv2/install/mymsg:/root/researchv2/install/laser_proc:/root/researchv2/install/diagnostic_common_diagnostics:/root/researchv2/install/chaser_system:/opt/ros/galactic LD_LIBRARY_PATH=/root/researchv2/install/diagnostic_aggregator/lib:/root/researchv2/install/urg_node/lib:/root/researchv2/install/urg_node_msgs/lib:/root/researchv2/install/urg_c/lib:/root/researchv2/install/mymsg/lib:/root/researchv2/install/laser_proc/lib:/opt/ros/galactic/opt/yaml_cpp_vendor/lib:/opt/ros/galactic/opt/rviz_ogre_vendor/lib:/opt/ros/galactic/lib/x86_64-linux-gnu:/opt/ros/galactic/lib:/usr/local/11.6.0/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64 PYTHONPATH=/root/researchv2/install/diagnostic_updater/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_aggregator/lib/python3.8/site-packages:/root/researchv2/install/urg_node_msgs/lib/python3.8/site-packages:/root/researchv2/install/chaser_system/lib/python3.8/site-packages:/root/researchv2/install/mymsg/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_common_diagnostics/lib/python3.8/site-packages:/opt/ros/galactic/lib/python3.8/site-packages /usr/bin/cmake --build /root/researchv2/build/diagnostics -- -j12 -l12
[0.094s] Invoked command in '/root/researchv2/build/diagnostics' returned '0': CMAKE_PREFIX_PATH=/root/researchv2/install/self_test:/root/researchv2/install/diagnostic_updater:/root/researchv2/install/diagnostic_aggregator:/root/researchv2/install/urg_node:/root/researchv2/install/urg_node_msgs:/root/researchv2/install/urg_c:/root/researchv2/install/diagnostics:/root/researchv2/install/mymsg:/root/researchv2/install/laser_proc:/root/researchv2/install/diagnostic_common_diagnostics:/root/researchv2/install/chaser_system:/opt/ros/galactic LD_LIBRARY_PATH=/root/researchv2/install/diagnostic_aggregator/lib:/root/researchv2/install/urg_node/lib:/root/researchv2/install/urg_node_msgs/lib:/root/researchv2/install/urg_c/lib:/root/researchv2/install/mymsg/lib:/root/researchv2/install/laser_proc/lib:/opt/ros/galactic/opt/yaml_cpp_vendor/lib:/opt/ros/galactic/opt/rviz_ogre_vendor/lib:/opt/ros/galactic/lib/x86_64-linux-gnu:/opt/ros/galactic/lib:/usr/local/11.6.0/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64 PYTHONPATH=/root/researchv2/install/diagnostic_updater/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_aggregator/lib/python3.8/site-packages:/root/researchv2/install/urg_node_msgs/lib/python3.8/site-packages:/root/researchv2/install/chaser_system/lib/python3.8/site-packages:/root/researchv2/install/mymsg/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_common_diagnostics/lib/python3.8/site-packages:/opt/ros/galactic/lib/python3.8/site-packages /usr/bin/cmake --build /root/researchv2/build/diagnostics -- -j12 -l12
[0.146s] Invoking command in '/root/researchv2/build/diagnostics': CMAKE_PREFIX_PATH=/root/researchv2/install/self_test:/root/researchv2/install/diagnostic_updater:/root/researchv2/install/diagnostic_aggregator:/root/researchv2/install/urg_node:/root/researchv2/install/urg_node_msgs:/root/researchv2/install/urg_c:/root/researchv2/install/diagnostics:/root/researchv2/install/mymsg:/root/researchv2/install/laser_proc:/root/researchv2/install/diagnostic_common_diagnostics:/root/researchv2/install/chaser_system:/opt/ros/galactic LD_LIBRARY_PATH=/root/researchv2/install/diagnostic_aggregator/lib:/root/researchv2/install/urg_node/lib:/root/researchv2/install/urg_node_msgs/lib:/root/researchv2/install/urg_c/lib:/root/researchv2/install/mymsg/lib:/root/researchv2/install/laser_proc/lib:/opt/ros/galactic/opt/yaml_cpp_vendor/lib:/opt/ros/galactic/opt/rviz_ogre_vendor/lib:/opt/ros/galactic/lib/x86_64-linux-gnu:/opt/ros/galactic/lib:/usr/local/11.6.0/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64 PYTHONPATH=/root/researchv2/install/diagnostic_updater/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_aggregator/lib/python3.8/site-packages:/root/researchv2/install/urg_node_msgs/lib/python3.8/site-packages:/root/researchv2/install/chaser_system/lib/python3.8/site-packages:/root/researchv2/install/mymsg/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_common_diagnostics/lib/python3.8/site-packages:/opt/ros/galactic/lib/python3.8/site-packages /usr/bin/cmake --install /root/researchv2/build/diagnostics
[0.156s] -- Install configuration: ""
[0.156s] -- Up-to-date: /root/researchv2/install/diagnostics/share/ament_index/resource_index/package_run_dependencies/diagnostics
[0.157s] -- Up-to-date: /root/researchv2/install/diagnostics/share/ament_index/resource_index/parent_prefix_path/diagnostics
[0.158s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/environment/ament_prefix_path.sh
[0.160s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/environment/ament_prefix_path.dsv
[0.161s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/environment/path.sh
[0.162s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/environment/path.dsv
[0.163s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/local_setup.bash
[0.163s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/local_setup.sh
[0.165s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/local_setup.zsh
[0.166s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/local_setup.dsv
[0.167s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/package.dsv
[0.168s] -- Up-to-date: /root/researchv2/install/diagnostics/share/ament_index/resource_index/packages/diagnostics
[0.168s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/cmake/diagnosticsConfig.cmake
[0.169s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/cmake/diagnosticsConfig-version.cmake
[0.170s] -- Up-to-date: /root/researchv2/install/diagnostics/share/diagnostics/package.xml
[0.190s] Invoked command in '/root/researchv2/build/diagnostics' returned '0': CMAKE_PREFIX_PATH=/root/researchv2/install/self_test:/root/researchv2/install/diagnostic_updater:/root/researchv2/install/diagnostic_aggregator:/root/researchv2/install/urg_node:/root/researchv2/install/urg_node_msgs:/root/researchv2/install/urg_c:/root/researchv2/install/diagnostics:/root/researchv2/install/mymsg:/root/researchv2/install/laser_proc:/root/researchv2/install/diagnostic_common_diagnostics:/root/researchv2/install/chaser_system:/opt/ros/galactic LD_LIBRARY_PATH=/root/researchv2/install/diagnostic_aggregator/lib:/root/researchv2/install/urg_node/lib:/root/researchv2/install/urg_node_msgs/lib:/root/researchv2/install/urg_c/lib:/root/researchv2/install/mymsg/lib:/root/researchv2/install/laser_proc/lib:/opt/ros/galactic/opt/yaml_cpp_vendor/lib:/opt/ros/galactic/opt/rviz_ogre_vendor/lib:/opt/ros/galactic/lib/x86_64-linux-gnu:/opt/ros/galactic/lib:/usr/local/11.6.0/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64 PYTHONPATH=/root/researchv2/install/diagnostic_updater/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_aggregator/lib/python3.8/site-packages:/root/researchv2/install/urg_node_msgs/lib/python3.8/site-packages:/root/researchv2/install/chaser_system/lib/python3.8/site-packages:/root/researchv2/install/mymsg/lib/python3.8/site-packages:/root/researchv2/install/diagnostic_common_diagnostics/lib/python3.8/site-packages:/opt/ros/galactic/lib/python3.8/site-packages /usr/bin/cmake --install /root/researchv2/build/diagnostics
